tests/foreman/ui/test_capsule.py

```diff
@@ -93,3 +93,349 @@ def test_positive_chost_selector(self):
 
         @Status: Manual
         """
+
+    @run_only_on('sat')
+    @stubbed
+    @tier1
+    def test_positive_capsule_version(self):
+        """Check Capsule Version in About Page.
+
+        @Feature: Capsule Management UX
+
+        @Steps:
+
+        1. Navigate to the About Page
+        2. Check the versions coloumn of the Capsules.
+
+        @Assert: Whether the version coloumn of the Capsules exist.
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed
+    @tier1
+    def test_positive_capsule_status(self):
+        """Check Capsule Status in Index Page.
+
+        @Feature: Capsule Management UX
+
+        @Steps:
+
+        1. Navigate to the Capsules Index Page
+        2. Check the status of the Proxy.
+
+        @Assert: Whether the status of the Capsules
```

Comment:
Actually this may not impact on docs generation. In that case the alignment here can be left as is. But for the code style sake, all other code align on the start instead of this way.